bin/build.js

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env node
+
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const path = require( 'path' );
+const webpack = require( 'webpack' );
+const { logger, bundler } = require( '@ckeditor/ckeditor5-dev-utils' );
+const webpackUtils = require( '@ckeditor/ckeditor5-dev-webpack-utils' );
+const buildConfig = require( '../build-config' );
+const log = logger();
+const entryPoint = 'ckeditor.js';
+
+log.info( 'Creating an entry file...' );
+
+bundler.createEntryFile( entryPoint, {
+	plugins: buildConfig.plugins,
+	moduleName: buildConfig.moduleName,
+	editor: buildConfig.editor,
+	config: buildConfig.editorConfig,
+} );
+
+const packageRoot = path.join( __dirname, '..' );
+const ckeditor5Root = path.join( packageRoot, '..', '..' );
+const webpackParams = {
+	cwd: ckeditor5Root,
+	entryPoint: path.join( packageRoot, entryPoint ),
+	destinationPath: path.join( packageRoot, buildConfig.destinationPath )
+};
+const webpackConfig = webpackUtils.getWebpackConfig( webpackParams );
+const webpackCompactConfig = webpackUtils.getWebpackCompactConfig( webpackParams );
+
+log.info( `Creating the "ES6" and "Compact" builds...` );
```

Comment:
Besides, you're not creating anything here. You're building the editor. So you can just log "Building..." and is enough.